docs/reference-metrics-opentelemetry.md

```diff
@@ -478,6 +478,120 @@ The total number of messages dropped from this producer.
+* Type: Counter
+* Unit: `{message}`
+* Attributes:
+  * `pulsar.domain` - The domain of the topic. Can be one of:
+    * `persistent`
+    * `non-persistent`
+  * `pulsar.tenant` - The topic tenant.
+  * `pulsar.namespace` - The topic namespace.
+  * `pulsar.topic` - The topic name.
+  * `pulsar.partition.index` - The partition index of the topic. Present only if the topic is partitioned.
+  * `pulsar.replication.remote.cluster.name` - The name of the remote cluster.
+
+#### pulsar.broker.replication.message.incoming.size
+The total number of messages bytes received from the remote cluster through this replicator.
+* Type: Counter
+* Unit: `{By}`
+* Attributes:
+  * `pulsar.domain` - The domain of the topic. Can be one of:
+    * `persistent`
+    * `non-persistent`
+  * `pulsar.tenant` - The topic tenant.
+  * `pulsar.namespace` - The topic namespace.
+  * `pulsar.topic` - The topic name.
+  * `pulsar.partition.index` - The partition index of the topic. Present only if the topic is partitioned.
+  * `pulsar.replication.remote.cluster.name` - The name of the remote cluster.
+
+#### pulsar.broker.replication.message.outgoing.size
+The total number of messages bytes sent to the remote cluster through this replicator.
+* Type: Counter
+* Unit: `{By}`
+* Attributes:
+  * `pulsar.domain` - The domain of the topic. Can be one of:
+    * `persistent`
+    * `non-persistent`
+  * `pulsar.tenant` - The topic tenant.
+  * `pulsar.namespace` - The topic namespace.
+  * `pulsar.topic` - The topic name.
+  * `pulsar.partition.index` - The partition index of the topic. Present only if the topic is partitioned.
+  * `pulsar.replication.remote.cluster.name` - The name of the remote cluster.
+
+#### pulsar.broker.replication.message.backlog.count
+The total number of messages in the backlog for this replicator.
+* Type: Counter
+* Unit: `{message}`
+* Attributes:
+  * `pulsar.domain` - The domain of the topic. Can be one of:
+    * `persistent`
+    * `non-persistent`
+  * `pulsar.tenant` - The topic tenant.
+  * `pulsar.namespace` - The topic namespace.
+  * `pulsar.topic` - The topic name.
+  * `pulsar.partition.index` - The partition index of the topic. Present only if the topic is partitioned.
+  * `pulsar.replication.remote.cluster.name` - The name of the remote cluster.
+
+#### pulsar.broker.replication.message.backlog.age
+The age of the oldest message in the replicator backlog.
+* Type: Gauge
+* Unit: `s`
+* Attributes:
+  * `pulsar.domain` - The domain of the topic. Can be one of:
+    * `persistent`
+    * `non-persistent`
+  * `pulsar.tenant` - The topic tenant.
+  * `pulsar.namespace` - The topic namespace.
+  * `pulsar.topic` - The topic name.
+  * `pulsar.partition.index` - The partition index of the topic. Present only if the topic is partitioned.
+  * `pulsar.replication.remote.cluster.name` - The name of the remote cluster.
+
+#### pulsar.broker.replication.message.expired.count
+The total number of messages that expired for this replicator.
```

Comment:
Can we clarify a little more here? something like

The total number of messages that expired for this replicator before consuming.